addons/web/static/src/legacy/scss/primary_variables.scss

```diff
@@ -114,6 +114,7 @@ $o-modal-md: 650px !default;
 $o-sheet-cancel-tpadding: 0px !default;
 
 $o-avatar-size: 90px !default;
+$o-avatar-inline-size: 1.4em !default;
```

Comment:
Is `1.4em` coming from a computation, or it's just an arbitrary value picked to look good?